aten/src/ATen/native/native_functions.yaml

```diff
@@ -4238,6 +4238,12 @@
     CUDA: _weight_int8pack_mm_cuda
     MPS: _weight_int8pack_mm_mps
 
+# Currently, we don't have fp8 pack, but we wish to align the name with int8
+# So that it is easier for the user to understand the API for WoQ case.
+- func: _weight_fp8pack_mm(Tensor self, Tensor mat2, Tensor? scales) -> Tensor
+  dispatch:
+    XPU: _weight_fp8pack_mm_xpu
```

Comment:
@drisspg , may I know your input on adding this operation to accelerate the performance of the case mentioned in the PR description?